quickstart/101-azure-virtual-desktop/options/netapp/netappstorage.tf

```diff
@@ -0,0 +1,67 @@
+resource "azurerm_subnet" "netapp_subnet" {
```

Comment:
What is the purpose of this options/netapp folder and contents? Is it supposed to be a sub module? If so, where is it referenced? I do not see it referenced in the docs either.

If the intent is for this to be an alternate to the afstorage.tf, you should create a second quickstart that uses this instead.

Reply:
I see the confusion. I created a second quickstart for the alternate storage options. Thank you.